storage/diemdb/src/lib.rs

```diff
@@ -886,6 +958,12 @@ impl DbWriter for DiemDB {
     }
 }
 
+impl Drop for DiemDB {
+    fn drop(&mut self) {
+        self.rocksdb_property_reporter.stop();
```

Comment:
why not just impl Drop for the reporter?

Reply:
👍